src/languages/en.js

```diff
@@ -49,6 +49,7 @@ export default {
         invite: 'Invite',
         iAcceptThe: 'I accept the ',
         passwordCannotBeBlank: 'Password cannot be blank',
+        send: 'Send',
```

Comment:
Can you also make sure these updates are included in the `es.js` file? You can just google translate to Spanish for now